src/index.js

```diff
@@ -2,6 +2,8 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import App from './App';
 
+export * from './components/Sum/Sum';
```

Comment:
suggestion: you'd rather create `index.js` file in `Sum` dir and put this line there
that's a common practice